src/main/java/Task.java

```diff
@@ -0,0 +1,101 @@
+import java.lang.invoke.SerializedLambda;
+
+public class Task {
+    protected static Task[] tasks;
+    public static int totalTasks = 0;
+    protected String description;
+    protected boolean isDone;
+
+    protected String type;
```

Comment:
What is this 'type' used for? I may be wrong, but I don't see where this variable is being used.

Reply:
Oh yes you're not wrong, it's not used for anything anymore. I forgot to remove it but basically previously I used type to store whether the object is Task, Deadline, or Event